PIVX-Labs/MyPIVXWallet
# Add logout button and encourage encryption

## Abstract

Add a log out button ( where log out of course means also remove the account from the database) and explicitly ask users to encrypt their wallet on creation/non-view import.
With the new wallet model we want to minimize the number of "first sync" (PR #219) that might take a while. If the wallet is encrypted txs can be saved and fetched on DB (PR for this will come soon), reducing the number of "first sync" to 1.

---

## Testing
- Verify that log out works fine 
- Verify that encryption is asked on wallet creation/import
